profitbricks-rest/src/main/java/org/apache/jclouds/profitbricks/rest/binder/BaseProfitBricksRequestBinder.java

```diff
@@ -16,20 +16,18 @@
  */
 package org.apache.jclouds.profitbricks.rest.binder;
 
-import static com.google.common.base.Preconditions.checkNotNull;
-
 import java.util.Map;
-
 import org.jclouds.http.HttpRequest;
 import org.jclouds.rest.MapBinder;
-
 import com.google.common.base.Strings;
-
 import org.jclouds.io.MutableContentMetadata;
 import org.jclouds.io.payloads.BaseMutableContentMetadata;
+import static com.google.common.base.Preconditions.checkNotNull;
 
 public abstract class BaseProfitBricksRequestBinder<T> implements MapBinder {
 
+   protected final String endpointPath = "/rest/v2";
```

Comment:
Yes, I'm trying to run the binder test. `UpdateImageRequestBinderTest.java`. To achieve this, I have 

`Injector injector = Guice.createInjector(new GsonModule(), new LocationModule());`. 

However I still get an error:
 ```20) No implementation for java.lang.Long annotated with @com.google.inject.name.Named(value=jclouds.session-interval) was bound.
  at org.jclouds.location.config.LocationModule.zoneIdsSupplier(LocationModule.java:128)```